test/no-unnecessary-polyfills.mjs

```diff
@@ -0,0 +1,182 @@
+		},
+		{
+			code: 'require("object-assign")',
+			options: [{targets: 'node <4'}],
+		},
+		{
+			code: 'require("object-assign")',
+			options: [{targets: 'node >3'}],
+		},
+		{
+			code: 'require()',
+			options: [{targets: 'node >3'}],
+		},
+		{
+			code: 'import("")',
+			options: [{targets: 'node >3'}],
+		},
+		{
+			code: 'import(null)',
+			options: [{targets: 'node >3'}],
+		},
+		{
+			code: 'require(null)',
+			options: [{targets: 'node >3'}],
+		},
+		{
+			code: 'require("" )',
+			options: [{targets: 'node >3'}],
+		},
+	],
+	invalid: [
+		{
+			code: 'require("setprototypeof")',
+			options: [{targets: 'node >4'}],
+			errors: [{message: 'Use the built-in `Object#setPrototypeOf`.'}],
+		},
+		{
+			code: 'require("core-js/features/array/last-index-of")',
+			options: [{targets: 'node >6.5'}],
+			errors: [{message: 'Use the built-in `Array#lastIndexOf`.'}],
+		},
+		{
+			code: 'require("core-js-pure/features/array/from")',
+			options: [{targets: 'node >7'}],
+			errors: [{message: 'All polyfilled features imported from `core-js-pure/features/array/from` are disponible as built-ins. Use the built-ins instead.'}],
+		},
+		{
+			code: 'require("core-js/features/array/from")',
+			options: [{targets: 'node >7'}],
+			errors: [{message: 'All polyfilled features imported from `core-js/features/array/from` are disponible as built-ins. Use the built-ins instead.'}],
+		},
+		// The feature esnext.typed-array.from-async polyfilled in this modules has not compat data
+		// {
+		// 	code: 'require("core-js/features/typed-array")',
+		// 	options: [{targets: 'node >16'}],
+		// 	errors: [{message: 'All polyfilled features imported from `core-js/features/typed-array` are disponible as built-ins. Use the built-ins instead.'}],
+		// },
+		{
+			code: 'require("es6-symbol")',
+			options: [{targets: 'node >15'}],
+			errors: [{message: 'Use the built-in `Symbol`.'}],
+		},
+		{
+			code: 'require("code-point-at")',
+			options: [{targets: 'node >4'}],
+			errors: [{message: 'Use the built-in `String#codePointAt`.'}],
+		},
+		{
+			code: 'require("object.getownpropertydescriptors")',
+			options: [{targets: 'node >8'}],
+			errors: [{message: 'Use the built-in `Object#getOwnPropertyDescriptors`.'}],
+		},
+		{
+			code: 'require("string.prototype.padstart")',
+			options: [{targets: 'node >8'}],
+			errors: [{message: 'Use the built-in `String#padStart`.'}],
+
+		},
+		{
+			code: 'require("p-finally")',
+			options: [{targets: 'node >10.4'}],
+			errors: [{message: 'Use the built-in `Promise#finally`.'}],
+
+		},
+		{
+			code: 'require("promise-polyfill")',
+			options: [{targets: 'node >15'}],
+			errors: [{message: 'Use the built-in `Promise`.'}],
+		},
+		{
+			code: 'require("es6-promise")',
+			options: [{targets: 'node >15'}],
+			errors: [{message: 'Use the built-in `Promise`.'}],
+		},
+		{
+			code: 'require("object-assign")',
+			options: [{targets: 'node 6'}],
+			errors: [{message: 'Use the built-in `Object#assign`.'}],
```

Comment:
The message should be ``'Use the built-in `Object.assign` ``, it's a static method.